packages/flutter/lib/src/widgets/container.dart

```diff
@@ -2,6 +2,10 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// ignore_for_file: prefer_final_locals, always_specify_types
+// Ignoring these rules helps to declutter the below switch expressions,
+// although reinstating prefer_final_locals could give a small performance boost.
```

Comment:
Please don't do this. We have these lints enabled for a reason.

Reply:
thank you, removed now.